src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" metod="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label for="fsurname" class="label">
+          Surname:
+          <input
+            type="text"
+            name="fsurname"
+            id="fsurname"
+            autocomplete="off"
+          >
+        </label>
```

Comment:
If you wrap `input` with` label` you do not need to use for/id
Apply all over the document
```suggestion
        <label class="label">
          Surname:
          <input
            type="text"
            name="fsurname"
            autocomplete="off"
          >
        </label>
```